### ISSUE TITLE
A11y_Azure Data Studio Nov2023_Server_Server Properties Preview_Use of color: Required fields in "Server Properties Preview" dialog is not associated with "*"(Star).

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#25159]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #WCAG3.3.2; #Labels or Instructions; #FTP; #SH_Azure Data Studio_Nov2023_Mac_Dec2023; #SQL Azure Data Studio; #MAC; 

### Environment Details:
Application Name: Azure Data Studio
Operating System: macOS Sonoma 14.2.1
Azure data studio 1.47.0

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Tab till "Server" and press F10 key to open the context menu.
4. Tab till "Properties Preview" and press enter.
5. "Server Properties Preview" dialog will appear.
6. Tab till "Memory" tab and press enter.
7. You will observe that Voiceover will announce the fields as required but no "*"(Start) is associated with fields nor any note is provided to fields as it's required.

### Actual Result:
Required fields in "Server Properties Preview" dialog is not associated with "*"(Start). Neither any note is provided to fields as it's required.

Note: Similar issue is observed for required fields throughout the application.

### Expected Result:
Required fields in "Server Properties Preview" dialog should be associated with "*"(Start) or note needs to be provided to fields as it's required if all the fields in dialog are required.

### User Impact:
Sighted-users will not able to unterstand and identify the form fields as required or not if there is no any chevron is associated.

### Attachment:
![Use of color](https://github.com/microsoft/azuredatastudio/assets/155441466/fcf40ab5-121e-4ef8-b75b-ef512c19735a)

**Comment by corivera** [comment]
@Mana-swi This has been fixed in the latest Insiders build for Mac [here](https://azuredatastudio-update.azurewebsites.net/latest/darwin-universal/stable). Can you verify the issue is fixed?

**Comment by Mana-swi** [comment]
Hi @corivera we are unable to download the latest Insiders build for Mac from provided link above. Can you please reshare it or help us with the build details. Thanks!

**Comment by Mana-swi** [comment]
#Regressed:02-05-24; Reopened;

Hi @corivera 
Verified the issue in below environment
Azure data Studio Version: 1.47.1 (Universal)
VS Code: 1.82.0
Electron: 25.8.4
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0

The issue is still repro'ing, please find below attachment for reference. Thanks!
![25159_Required fields in Server Properties Preview dialog is not associated with Star](https://github.com/microsoft/azuredatastudio/assets/155441466/3c808e49-dbca-47eb-92e5-5849ccfd692a)

**Comment by barbaravaldez** [comment]
@Mana-swi, I checked on latest insiders build and it seems to be fixed. It looks like you are using the latest release and not the insiders build: https://azuredatastudio-update.azurewebsites.net/latest/darwin-universal/insider

**Comment by Mana-swi** [comment]
#Regressed:02-06-24; #Closed;

Reverified the issue in below environment
Version: 1.48.0-insider (Universal)
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
MacOS: sonoma 14.2.1

The issue is fixed, hence closing the bug. Please find attachment for reference. Thanks!
![Star mark for required fields](https://github.com/microsoft/azuredatastudio/assets/155441466/dd99ffcb-39ec-4463-b876-b57d0db3bdad)
